README.md

```diff
@@ -86,7 +86,7 @@ Register a domain `$DOMAIN` with desec.io, and obtain a DNS management token `$T
     EMAIL=youremail@example.com
     echo "dns_desec_token = $TOKEN" > desec-secret.ini
     chmod 600 desec-secret.ini
-    certbot \
+    ./venv/bin/certbot \
```

Comment:
I don't think this is a clarification. First, the virtual environment is not necessarily located in `./venv`. Second, if it is activated, it's in `$PATH` and a path needs not be specified.

That said, if you'd like to have something like that, I'd suggest to make the creation of the `venv`  directory explicit (as a command). Then it's clear that it only works when done like that.

Reply:
On my machine, activating the virtual environment didn't update the `$PATH` accordingly (or with high enough priority). I'll update the README to include the virtual environment directory name though.

Reply:
Hm. At least, as far as I remember, when I activate a venv, `which python` points to the venv's Python executable. I had concluded from that that it must be in the `$PATH`.

Reply:
I can't immediately reproduce the problematic behavior on my machine. However, with the well-defined venv path I currently don't see a downside in giving the potentially over-specific example.